lib/shelley/test/data/cardano-node-shelley/node.config

```diff
@@ -30,9 +30,12 @@ LastKnownBlockVersion-Major: 2
 LastKnownBlockVersion-Minor: 0
 LastKnownBlockVersion-Alt: 0
 
-# Launcher code will submit an update proposal to trigger the hard-fork.
-TestShelleyHardForkAtEpoch: 1
-TestAllegraHardForkAtEpoch: 3
+# These settings start the test cluster in the Mary era (a "virtual"
+# hard fork happens at the start of the first epoch).
+# They will be generated according to the local test cluster config.
```

Comment:
Do we need that in this PR? Can't we stick to having this hard-coded in the configuration for now and adds this feature later?

Reply:
Started a related [slack thread](https://input-output-rnd.slack.com/archives/GBT05825V/p1610379419068700)

Reply:
I think we need it, because of all the AnyCardanoEra cases which are being changed. In this PR? Well, it's testing the new code from this PR.

But one thing led to another, and a small patch on the test cluster spread out a bit. So I have removed those test cluster era changes and put them into PR #2438.